src/modules/asyncButtons/i18n/sv_SE.json

```diff
@@ -0,0 +1,37 @@
+{
+	"deleteARR": {
+		"deleteModal": {
+			"btnCancel": "Avbryt",
+			"btnConfirm": "Radera",
+			"text": "Vill du radera denna AAO-post?",
+			"title": "Radera?"
+		}
+	},
+	"forumPost": {
+		"deleteModal": {
+			"btnCancel": "Avbryt",
+			"btnConfirm": "Radera",
+			"text": "Vill du radera det här inlägget?",
+			"title": "Radera?"
+		}
+	},
+	"memberlistManageUser": {
+		"hide": "Stäng",
+		"kickModal": {
+			"btnCancel": "Avbryt",
+			"btnConfirm": "Kick",
+			"text": "Vill du verkligen sparka ut spelaren?",
+			"title": "Kick?"
+		},
+		"set": "set",
+		"unset": "ta bort"
+	},
+	"memberlistRoles": {
+		"admin": "Alliance Administratör",
+		"aufsichtsrat": "Moderator-logg",
+		"coadmin": "Alliance Vice administratör",
+		"finance": "Alliance Ekonomiansvarig",
+		"schooling": "Alliance Utbildare",
+		"sprechwunsch_admin": "Transportansvarig-Administratör"
```

Comment:
![grafik](https://user-images.githubusercontent.com/27238105/101630948-d95dc680-3a23-11eb-91c7-694c7c761849.png)
exact the ones listed in here please!